### ISSUE TITLE
Only batchsize=1?

### ISSUE THREAD

**Issue by gjd2017** [multimodallearning/pytorch-mask-rcnn#40]
Hi, the code dose not support batchsize>1?

**Comment by jytime** [comment]
Hi @gjd2017 , have you found a pytorch mask rcnn implementation that supports batchsize>1? Thanks a lot

**Comment by gjd2017** [comment]
No, I use a for loop to get featuremaps(batchsize> 1), but something was wrong with results. You can try it.




------------------ 原始邮件 ------------------
发件人: "Jianyuan"<notifications@github.com>;
发送时间: 2018年9月6日(星期四) 中午1:58
收件人: "multimodallearning/pytorch-mask-rcnn"<pytorch-mask-rcnn@noreply.github.com>;
抄送: "幽谷"<1414253285@qq.com>; "Mention"<mention@noreply.github.com>; 
主题: Re: [multimodallearning/pytorch-mask-rcnn] Only batchsize=1? (#40)




Hi @gjd2017 , have you found a pytorch mask rcnn implementation that supports batchsize>1? Thanks a lot

**Comment by jytime** [comment]
@gjd2017 I have tried to expand the batch size and the model seems to work well. Where does your model goes wrong?

**Comment by jytime** [comment]
@gjd2017 I have put my implementation [here](https://github.com/jytime/Mask_RCNN_Pytorch), if you need

**Comment by Britefury** [comment]
I have created a fork in which I have made it work with batch size > 1, available here:

https://github.com/Britefury/pytorch-mask-rcnn/tree/refactor

(refactor branch).

I think that I have changed the API a lot, so many of the example programs wont work.

**Comment by gjd2017** [comment]
Thank you ! Nice!




------------------ 原始邮件 ------------------
发件人: "Britefury"<notifications@github.com>;
发送时间: 2018年9月12日(星期三) 下午4:35
收件人: "multimodallearning/pytorch-mask-rcnn"<pytorch-mask-rcnn@noreply.github.com>;
抄送: "幽谷"<1414253285@qq.com>; "Mention"<mention@noreply.github.com>; 
主题: Re: [multimodallearning/pytorch-mask-rcnn] Only batchsize=1? (#40)




I have created a fork in which I have made it work with batch size > 1, available here:
 
https://github.com/Britefury/pytorch-mask-rcnn/tree/refactor
 
(refactor branch).
 
I think that I have changed the API a lot, so many of the example programs wont work.
